src/mobile/apps/show/test/helpers/view_helpers.test.coffee

```diff
@@ -0,0 +1,33 @@
+sinon = require 'sinon'
+Show = require '../../../../models/show'
+{ fabricate } = require 'antigravity'
+ViewHelper = require '../../helpers/view_helpers.coffee'
+
+describe 'toJSONLD', ->
+  context 'when show has location', ->
+    show = new Show fabricate 'show'
```

Comment:
This pattern (creating a variable here and using it throughout the tests) worries me a _little_ bit because it would be easy for a test to mutate this variable and have it pollute other assertions.

A better pattern here could be to have a `beforeEach` block that creates a variable that can be referenced in the tests and cleaned up after each one.

See [this](https://github.com/artsy/force/blob/master/src/desktop/test/models/sale.test.coffee#L19) for an example of what I mean.